client/src/components/Modal/index.js

```diff
@@ -0,0 +1,29 @@
+import "./modal.css";
+import Component from "@core/Component";
+import { div } from "@core/CreateDom";
+import { ModalState } from "@store/";
+import CategoryDeleteForm from "./CategoryDeleteForm";
+import CategoryAddForm from "./CategoryAddForm";
+
+const modalMap = {
+    delete: CategoryDeleteForm,
+    add: CategoryAddForm,
+};
+
+export default class Modal extends Component {
+    bindState() {
+        return [ModalState];
+    }
+
+    render() {
+        console.log("modal");
+        const { value, type } = ModalState;
+        // prettier-ignore
+        return  type ? (div({class:"modalContainer"})(
+            div({class:"blackScreen"})(),
+            div({class:"modalWrapper"})(
+                modalMap[type](value)    
+            ))
+        ) :div();
```

Comment:
```suggestion
        if (!type) {
            return div();
        }
        return  (div({class:"modalContainer"})(
            div({class:"blackScreen"})(),
            div({class:"modalWrapper"})(
                modalMap[type](value)    
            ))
        );
```

와 같은 형태로 `earlyReturn` 을 사용하시는 것은 어떤가요?

Reply:
좋습니다!!! 이게 훨씬 깔끔하네요!